src/software/thunderscope/replay/replay_constants.py

```diff
@@ -0,0 +1,7 @@
+REPLAY_METADATA_DELIMETER = ","
+REPLAY_FILE_EXTENSION = "replay"
+REPLAY_FILE_TIME_FORMAT = "%Y_%m_%d_%H%M%S"
+REPLAY_MAX_CHUNK_SIZE_BYTES = 1024 * 1024 * 1  # 1 MiB
```

Comment:
Nit: * 1?

Reply:
yeah just thought the math was a bit more clear but it probably doesn't matter lol, removed